pom.xml

```diff
@@ -27,7 +27,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```

Comment:
This is needed for certain annotations such as Getters and Setters.

Reply:
Great job.